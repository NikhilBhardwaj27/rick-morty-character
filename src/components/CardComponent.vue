<template lang="">
    <div class=" border-black border flex flex-col p-5 gap-2 mb-3">
        <slot></slot>
    </div>
</template>
<script setup>
</script>
<style lang="">
    
</style>
