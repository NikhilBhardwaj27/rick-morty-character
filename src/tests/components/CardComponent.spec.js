import {mount} from "@vue/test-utils";
import CardComponent from '../../components/CardComponent.vue';


it("snapshot UI testing of CardComponent.vue", async () => {
    const wrapper = mount(CardComponent);
    expect(wrapper.text()).toMatchSnapshot()
});