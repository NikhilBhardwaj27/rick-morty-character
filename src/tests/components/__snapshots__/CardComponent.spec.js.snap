// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`snapshot UI testing of CardComponent.vue 1`] = `""`;
